docs/en/workbench/how_to/create_workbench.mdx

```diff
@@ -0,0 +1,58 @@
+---
+weight: 14
+---
+
+# Create Workbench
+
+:::note
+1. `WorkspaceKind` is the resource that cluster-admins create.
+2. `WorkspaceKind` specifies the template for a Workspace (e.g. "JupyterLab", "VSCode", "RStudio (in the feature)").
+:::
+
+## Prerequisites
+
+- Ensure you have `kubectl` configured and connected to your cluster.
+- Ensure you have created `WorkspaceKind` (cluster resource).
+
+:::tip
+When use `jupyterlab_v1beta1_workspacekind.yaml`, please change `CLUSTER_NAME` in `NB_FREIX` to your target cluster name, which is WorkspaceKind will be created at.
+:::
```

Comment:
IMPORTANT: when creating `workspacekind`, image URL must be changed to the actuall one for the currnet local cluster !!!